doc/circuitikzmanual.tex

```diff
@@ -441,7 +441,7 @@ \subsection{Getting started with \Circuitikz: a current shunt}
 
 Let's start with the first type of component, and build a basic mesh:
 
-\begin{ LTXexample}[varwidth=true]
+\begin{LTXexample}[varwidth=true]
```

Comment:
Uy! This is very strange because I haven't this space in my copy --- so maybe it has been added spuriously (notice that the commits are checked automatically by compiling the manual).

Reply:
I added it inadvertently in one of my revisions.  It took a while to track down the cause.

I will follow your suggestion of having fewer commits.  I was afraid overwhelming with a significant number of edits, but overwhelming with a significant number of commits isn't good, either.

I think everything is ready or merging, if the changes meet with your approval.

Improving the fluidity of the language is of interest to me, and as a native speaker of English I recognize what phrasing is good and what is not so good.  I am also willing to correct punctuation, but I didn't want to be a pain in the rear on this my first ever contribution via git to any project.  If you're willing to consider them, I'll include those in the revisions I propose.